var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -72,7 +73,7 @@ class Mvapich2(AutotoolsPackage):
     variant(
         'fabrics',
         description='The fabric enabled for this build',
-        default='psm',
+        default='mrail',
```

Comment:
Is this because OSU recommends `mrail`?  Which are users more likely to use?  I think whatever that is should likely be the default.

Reply:
ok sounds good!